### ISSUE TITLE
message to is too long to encrypt with given key length

### ISSUE THREAD

**Issue by jibanes** [burtonsamograd/rsa#3]
"rsa-encrypt-text: message to is too long to encrypt with given key length"

Shouldn't it really be cyclic and use padding appropriately?

**Comment by burtonsamograd** [comment]
Yes, I'll get to that eventually.  Right now it just encodes up to a the key size.

Patches welcome.
